CrackingTheCodingInterview/Chapter1ArraysAndStrings/Permutation.java

```diff
@@ -1,7 +1,13 @@
-// Given two strings, write a metho dto decide if one is a permutation of the other
+//Different examples of implementations of arrays and strings to determine permutations. 
 
 public class Permutation {
-	public boolean permutation(String s, String t) {
+	/**
+	*boolean method to compare two Strings and determine whether they are permutations of each other.
```

Comment:
I think you're over 100 chars here. Please split this comment into another line. 

Your comment formatting is also wrong: 
  - stars need to be aligned
  - cannot start method comment with lower case 
  - remove "boolean method to" --> "Compares two strings to determine whether they are permutations of each other."